controllers/ovndbcluster_controller.go

```diff
@@ -786,6 +805,70 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	return ctrl.Result{}, nil
 }
 
+// generateServiceConfigMaps - create create configmaps which hold service configuration
+func (r *OVNDBClusterReconciler) generateExternalConfigMaps(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *ovnv1.OVNDBCluster,
+	serviceName string,
+	envVars *map[string]env.Setter,
+) error {
+	// Create/update configmaps from templates
+	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(serviceName), map[string]string{})
+	encapType := ""
+
+	externalEndpoint, err := instance.GetExternalEndpoint()
+	if err != nil {
+		return err
+	}
+
+	ovnController, err := ovnv1.GetOVNController(ctx, h, instance.Namespace)
+	if k8s_errors.IsNotFound(err) {
+		// If ovnController is not found use geneve as default
+		encapType = "geneve"
```

Comment:
What should we do when OVNController is not spawned?
If there's no nicMappings configured, openstack-operator won't create any OVNController CR, so even if the OSCP has the encapType information, ovn-operator won't have the CR to get it, should we set geneve as default? Leave it empty?

Reply:
We do not need to set/pass it to edpm if there is no ovn controller on control plane side. ovn-controller on edpm nodes will keep using what configured there i.e with edpm_ovn_encap_type

Reply:
Yep, either we setup 'geneve' whenever ovnController is not set (geneve is the [default in the edpm_ansible](https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_ovn/defaults/main.yml#L41) ovn role) or we don't set up the encapType at all, which will need a modification on how the configMap is generated.
Whatever is set here will overwrite the default on edpm_ansible (even if it's empty)